lib/dpul_collections_web/components/core_components.ex

```diff
@@ -729,10 +733,10 @@ defmodule DpulCollectionsWeb.CoreComponents do
     ~H"""
     <header class={[@actions != [] && "flex items-center justify-between gap-6", @class]}>
       <div>
-        <h1 class="text-lg font-semibold leading-8 text-zinc-800">
+        <h1 class="font-bold">
           {render_slot(@inner_block)}
         </h1>
-        <p :if={@subtitle != []} class="mt-2 text-sm leading-6 text-zinc-600">
+        <p :if={@subtitle != []} class="mt-2 text-lg leading-6 text-zinc-600">
```

Comment:
Not sure about this text color, we don't have it anywhere else

Reply:
Good catch, gone.